third_party/istio-stable

```diff
@@ -0,0 +1 @@
+istio-1.4-latest
```

Comment:
<!--whitespace-bot-->
Add trailing newline:
```suggestion
istio-1.4-latest

```

Reply:
It is a symlink. We can ignore this.